types/react-hyperscript-helpers.errors.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-unused-vars */
+import { ReactElement } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+
+// Invalid prop
+// THROWS Argument of type '{ invalidProp: boolean; }' is not assignable to parameter of type 'ReactNode[] | WithDataAttributes<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>> | undefined'
```

Comment:
I like that we are using a type checking tool here for errors and type checks.  Two thoughts:
1) Can we move the xxxx.errors.ts and xxxx.types.ts files into their /types/react-hyperscript-helpers/ folder.
2) Can we add a README to the types folder explaining the type checker too, doc link,  and example commands?